sasmodels/kerneldll.py

```diff
@@ -291,7 +303,7 @@ def make_dll(source, model_info, dtype=F64, system=False):
         # Comment the following to keep the generated C file.
         # Note: If there is a syntax error then compile raises an error
         # and the source file will not be deleted.
-        os.unlink(filename)
+        # os.unlink(filename)
```

Comment:
Put back the `unlink()`. This removes the intermediate C file that gets compiled to the .so file.

BTW, I noticed that there were a bunch of .def files in your compiled models directory. Presumably tcc put them there as part of building the dll. We could delete those as well since we are never linking against the model files.